index.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
     'plugin:prettier/recommended',
     'prettier/react',
   ],
+  parserOptions: {
+    ecmaVersion: 2020,
+  },
```

Comment:
Any background why? @WanderingBrooks

Reply:
I should've put the prs up at the same time but I just sent a message in software dev :)

Reply:
But its because I want to bump the node version

Reply:
It could be that it is repo specific though which parser it should use. So maybe this will be added only to pumbaa